php/src/class-plugin.php

```diff
@@ -51,12 +49,15 @@ class Plugin {
 	public function __construct( $dir_path ) {
 		$this->dir_path = rtrim( $dir_path, '\\/' );
 		$this->dir_url  = content_url( str_replace( WP_CONTENT_DIR, '', $this->dir_path ) );
+		$this->settings = new Plugin_Settings( $this );
```

Comment:
The plugin instance is not being used within `Plugin_Settings`.
```suggestion
		$this->settings = new Plugin_Settings();
```